alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+  && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
+  && /root/setup-directories.sh postgres rw /var/db/postgresql \
+  && ln -s -- /etc/sv/postgres /etc/service/postgres/run \
+  && ln -s -- /etc/sv/buildkite /etc/service/buildkite/run \
+  && echo '@testing https://s3.amazonaws.com/alpine-packages/testing' >> /etc/apk/repositories \
+  && apk add --update-cache imagemagick buildkite-agent@testing phantomjs@testing
```

Comment:
I think explicit dependencies should be specified in a way that is similar to how we did this for alpine-rails-nginx.